pkgs/top-level/all-packages.nix

```diff
@@ -17965,7 +17967,12 @@ with pkgs;
 
   drm_info = callPackage ../development/tools/drm_info { };
 
-  drush = callPackage ../development/tools/misc/drush { };
+  drush = throw (
```

Comment:
This should be moved to `aliases.nix`.

Reply:
But these are two different tools, do you really think we should move it to `aliases.nix` ?

Reply:
Throws for removed packages also go there.

Reply:
Fixed